Create bot boilerplate

interrrp opened the pull request:
This contains logging and extensions. Logs will be written to `zz.log`.

ethandhunt committed:
fix: add .vscode/ to gitignore

Commit:
feat: create bot boilerplate

unseeyou commented:
better than what I had since it's my first time using disnake

Beardrop2 reviewed (approved):
Yeah looks like a solid snake start.

Beardrop2 committed:
ci: Setup .env file

The CI environment wont have dev setup.
This line creates a .env file based on preconfigured env vars.

unseeyou reviewed (approved):
looks good
